lnrpc/lightning.proto

```diff
@@ -746,6 +746,12 @@ message GetTransactionsRequest {
 
     // An optional filter to only include transactions relevant to an account.
     string account = 3;
+
+    // An optional filter to include transactions with certain transaction
+    // hashes. To filter for multiple transactions, though REST, provide
+    // duplicate GET parameters
+    // (?filter_tx_hash=<hash1>&filter_tx_hash=<hash2>&...).
```

Comment:
Didn't know the REST proxy supported this feature... Did you actually test this? Very nice if that's indeed the case.

Reply:
Works for me.